src/components/common/TmConnectedNetwork.vue

```diff
@@ -24,23 +24,24 @@
           class="tm-connected-network__string"
         >
           <span v-tooltip.top="networkTooltip" class="chain-id">
-            {{ connection.network }}
+            {{ connection.lastHeader.chain_id }}
```

Comment:
What is the thinking here?

Reply:
the networks page shows `Cosmos Hub` (title) and `cosmoshub-2` (chain ID) but `connection.network` is a third way of displaying network info that I found confusing (`cosmoshub`). 

maybe it would be better to show the same title as from the networks page, but i felt the chain ID was more specific and in talking about the connection this felt more appropriate.

Reply:
Yeah I was getting at maybe we should show the title. We wanted to make the chain_id transparent for the user.

Reply:
we can do that. i'm happy with chain-id but title is ok too i think. this means we have to query networks from the connected component, right?

Reply:
yep, check `PageNetworks` on how to query the networks. But also feel free to merge this branch and change this in another PR.